dotCMS/src/integration-test/java/com/dotcms/scheduler/SchedulerAPITest.java

```diff
@@ -0,0 +1,100 @@
+package com.dotcms.scheduler;
+
+import java.util.Random;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UUIDGenerator;
+import com.github.kagkarlsson.scheduler.Scheduler;
+import com.github.kagkarlsson.scheduler.SchedulerName;
+import com.github.kagkarlsson.scheduler.task.helper.OneTimeTask;
+import com.github.kagkarlsson.scheduler.task.helper.Tasks;
+
+public class SchedulerAPITest {
+    
+    @CloseDBIfOpened
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+        new DotConnect().setSQL("delete from scheduled_tasks").loadResult();
+
+    }
+
+    private final OneTimeTask<DotTask> oneTime =
+                    Tasks.oneTime("one-time-task", DotTask.class).execute((inst, ctx) -> {
+                        inst.getData().runTask(inst.getId(), inst.getData());
+                    });
+    
+
+    
+    /**
+     * This method starts another scheduler (like one running on another server)- along with the
+     * scheduler embedded in the SchedulerAPI to ensure that DotSyncronized tasks run in series, one at
+     * a time
+     * 
+     * @throws InterruptedException
+     */
+    @Test
+    public void test_sync_task() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)